src/main/kotlin/org/rust/ide/intentions/GenerateDocstringStubIntention.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.psi.PsiElement
+import com.intellij.util.text.CharArrayUtil
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+import org.rust.lang.core.psi.impl.RsFunctionImpl
+import org.rust.lang.core.types.ty.Ty
+
+class GenerateDocStubIntention : RsElementBaseIntentionAction<GenerateDocStubIntention.Context>() {
```

Comment:
Don't forget to add the corresponding files:

* description.html
* before.rs.template
* after.rs.template

<img width="724" alt="Screenshot 2021-04-09 at 19 52 58" src="https://user-images.githubusercontent.com/6079006/114214630-34b9aa00-996d-11eb-9929-a06b4c3dd8e2.png">